dotnet-grad/Models/AddressModel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace dotnet_grad.Models
+{
+  public class AddressModel
+  {
+    public AddressModel(string line, string line2, string city, string state, int code)
+    {
+      this.line = line;
+      this.line2 = line2;
+      this.city = city;
+      this.state = state;
+      this.code = code;
+    }
+
+    [Key] public int id { get; set; }
```

Comment:
The Key is inferred with the member name "id" - not required to add the data annotation unless you have another name for the primary key of this object